modular_citadel/code/modules/food_and_drinks/recipes/tablecraft/recipes_misc.dm

```diff
@@ -0,0 +1,11 @@
+/datum/crafting_recipe/food/stabsinthe //You can make a bottle of stabsinthe with a full bottle of absinthe and a nettle
```

Comment:
The tablecrafting is a bit weird. It doesn't consume any of the required items, just spawning the result. It DOES move the absinthe from whatever container it was in into the new bottle (which already has 100 stabsinthe, resulting in 200u total). I have no idea why it happens like that!